src/components/Console.vue

```diff
@@ -0,0 +1 @@
+//Define new console component
```

Comment:
This is a good start, but we need some code in here!

Reply:
Gotcha!